language/de/install.php

```diff
@@ -381,12 +381,12 @@
 			<li>Aktualisiere deine Styles<br><br></li>
 		</ul>
 
-		<h1>Wie du ein Update deiner Installation mit dem „Automatischen-Update-Paket“ durchführst</h1>
+		<h1>Wie du ein Update deiner Installation mit dem „Erweiterte-Update-Paket“ durchführst</h1>
 
-		<p>Das „Automatische-Update-Paket“ wird nur in dem Fall empfohlen, wenn in deiner Installation die phpBB-Hauptdateien verändert wurden. Du kannst deine Installation auch mit den in der INSTALL.html beschriebenen Methoden aktualisieren. Zum automatischen Update von phpBB musst du folgende Schritte ausführen:</p>
+		<p>Das „Erweiterte-Update-Paket“ wird nur in dem Fall empfohlen, wenn in deiner Installation die phpBB-Hauptdateien verändert wurden. Du kannst deine Installation auch mit den in der INSTALL.html beschriebenen Methoden aktualisieren. Zum automatischen Update von phpBB musst du folgende Schritte ausführen:</p>
```

Comment:
Sorry, Korrektur: da fehlt eine Ersetzung:
```suggestion
		<p>Das „Erweiterte-Update-Paket“ wird nur in dem Fall empfohlen, wenn in deiner Installation die phpBB-Hauptdateien verändert wurden. Du kannst deine Installation auch mit den in der INSTALL.html beschriebenen Methoden aktualisieren. Zum erweiterten Update von phpBB musst du folgende Schritte ausführen:</p>
```